src/application/modules/legacy/module.ts

```diff
@@ -108,7 +108,10 @@ export class LegacyModule extends BaseModule {
 			moduleConfig,
 		) as ModuleConfigJSON;
 		this._moduleConfig = getModuleConfig(genesisConfig, mergedModuleConfig);
-		this._reclaimLSKCommand.init({ tokenIDReclaim: this._moduleConfig.tokenIDReclaim });
+		this._reclaimLSKCommand.init({
+			tokenIDReclaim: this._moduleConfig.tokenIDReclaim,
+			name: this.name,
```

Comment:
Please consider refactoring the property to be `moduleName` instead.
```suggestion
			moduleName: this.name,
```